File: app/src/main/java/com/jmalinen/blackjack/model/Suit.kt
package com.jmalinen.blackjack.model

enum class Suit(val symbol: Char, val isRed: Boolean) {
    HEARTS('\u2665', true),
    DIAMONDS('\u2666', true),
    CLUBS('\u2663', false),
    SPADES('\u2660', false)
}

File: ios/Blackjack/Blackjack/Model/Suit.swift
enum Suit: CaseIterable {
    case hearts
    case diamonds
    case clubs
    case spades

    var symbol: Character {
        switch self {
        case .hearts: "\u{2665}"
        case .diamonds: "\u{2666}"
        case .clubs: "\u{2663}"
        case .spades: "\u{2660}"
        }
    }

    var isRed: Bool {
        switch self {
        case .hearts, .diamonds: true
        case .clubs, .spades: false
        }
    }
}
